gists_database/search.py

```diff
@@ -1,5 +1,52 @@
 from .models import Gist
+from datetime import datetime
+
+
+def operator(op):
+    d = {'gt': '>', 'gte': '>=', 'lt': '<', 'lte': '<='}
+    if op in d:
+        return d[op]
+    return '=='
```

Comment:
👍 smart solution! For this, you could have used `d.get(op, '==')` to simplify it.